exonum-light-client/src/main/java/com/exonum/client/ExplorerApiHelper.java

```diff
@@ -44,18 +47,23 @@
  */
 final class ExplorerApiHelper {
 
+  @VisibleForTesting
+  static final Gson JSON = JsonSerializer.builder()
```

Comment:
Either create another `Gson` object in `SystemApiHelper` or remove `@VisibleForTesting`.

Reply:
Yes, ideally it must be in some static utility class… Will see if there is a good candidate.

Reply:
Decided to move to ExonumApi instead of adding a separate class to hold that. @bullet-tooth